WordPress/Classes/Utility/ImmuTableViewController.swift

```diff
@@ -5,7 +5,6 @@ import WordPressShared
 typealias ImmuTableRowControllerGenerator = ImmuTableRow -> UIViewController
 
 protocol ImmuTablePresenter: class {
-    var visible: Observable<Bool> { get }
     func push(controllerGenerator: ImmuTableRowControllerGenerator) -> ImmuTableAction
```

Comment:
Your previous version of this PR added a `present` method here, too. Should it be in this version?